pkg/controller/cleanup/cleanup.go

```diff
@@ -113,6 +114,19 @@ func (c *Controller) HandleCleanup() http.Handler {
 			c.logger.Infof("purged %v verification tokens", count)
 		}
 
+		// Gather usage statistics
+		if err := c.db.UpdateUserStats(); err != nil {
```

Comment:
I added these as part of cleanup. They are quick runs and not having it on all user page loads should avoid db contention as things scale.

This could be made less frequent, but updating the issued (1d) sort of needs frequent enough updates to be useful.